File: web-server/src/main/java/com/churchofphilippi/webserver/model/Event.java
package com.churchofphilippi.webserver.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.time.LocalDateTime;

@Entity
@Table(name = "event")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Event {

    @Id
            @SequenceGenerator(
            name = "event_sequence",
            sequenceName = "event_sequence",
            allocationSize = 1
    )
    @GeneratedValue(
            strategy = GenerationType.SEQUENCE,
            generator = "event_sequence"
    )
    @Column(
            name = "event_id",
            updatable = false
    )
    private Long eventId;

    @Column(nullable = false)
    private LocalDateTime startDate;

    @Column(nullable = false)
    private LocalDateTime endDate;

    @Column(nullable = false)
    private String title;

    @Column(nullable = false)
    private String color;

    @Column(nullable = false)
    private boolean allDay;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "dept_id")
    @JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
    private Dept department;

}

File: cop_mobile/cop_mobile/cop_mobile/Model/Event.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace cop_mobile.Model
{
    public class Event
    {
        public long eventId { get; set; }
        public DateTime startDate { get; set; }
        public DateTime endDate { get; set; }
        public string title { get; set; }
        public string color { get; set; }
        public bool allDay { get; set; }
        public Department department { get; set; }

        public Event(long eventId, DateTime startDate, DateTime endDate, string title, string color, bool allDay, Department department)
        {
            this.eventId = eventId;
            this.startDate = startDate;
            this.endDate = endDate;
            this.title = title;
            this.color = color;
            this.allDay = allDay;
            this.department = department;
        }
    }
}
